src/icons/fileIcons.ts

```diff
@@ -929,7 +930,6 @@ export const fileIcons: FileIcons = {
         '.eslintrc.yml',
         '.eslintrc.json',
         '.eslintrc-md.js',
-        '.eslintrc-jsdoc.js',
```

Comment:
Are you sure that you want to remove this file name again?

Reply:
Nice catch. It was a mistake, my friend :)